apps/judicial-system/web/src/routes/Court/InvestigationCase/CourtRecord/CourtRecord.tsx

```diff
@@ -511,17 +515,9 @@ const CourtRecord: FC = () => {
             handleNavigationTo(constants.INVESTIGATION_CASE_CONFIRMATION_ROUTE)
           }
           nextIsDisabled={!stepIsValid}
-          hideNextButton={
-            !workingCase.isCompletedWithoutRuling
-              ? !workingCase.decision ||
-                !workingCase.conclusion ||
-                !workingCase.ruling
-              : !workingCase.decision
-          }
+          hideNextButton={hasMissingInfoInRulingStep}
```

Comment:
This specifically targets users to check the previous page while the step is valid focuses on missing data in the current page as well